Come√ßando testes.. n
./startPython.sh: linha 2: virtualenv/bin/activate: Arquivo ou diret√≥rio inexistente
./startPython.sh: linha 3: pytest: comando n√£o encontrado
Os testes falharam.
Come√ßando testes.. n
./startPython.sh: linha 2: virtualenv/bin/activate: Arquivo ou diret√≥rio inexistente
./startPython.sh: linha 3: pytest: comando n√£o encontrado
Os testes falharam.
Come√ßando testes.. n
./startPython.sh: linha 2: virtualenv/bin/activate: Arquivo ou diret√≥rio inexistente
./startPython.sh: linha 3: pytest: comando n√£o encontrado
Os testes falharam.
Come√ßando testes.. n
./startPython.sh: linha 2: virtualenv/bin/activate: Arquivo ou diret√≥rio inexistente
./startPython.sh: linha 3: pytest: comando n√£o encontrado
Os testes falharam.
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 315-855-487
127.0.0.1 - - [23/Jun/2025 13:58:04] "GET /check HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 13:58:11] "GET /images/Cloro.jpeg HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 13:58:52] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 13:58:52] "[36mGET /static/index.css HTTP/1.1[0m" 304 -
127.0.0.1 - - [23/Jun/2025 13:58:52] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [23/Jun/2025 13:58:54] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 13:58:54] "[36mGET /static/index.css HTTP/1.1[0m" 304 -
127.0.0.1 - - [23/Jun/2025 13:59:26] "GET /imageGallery HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 13:59:26] "[35m[1mGET /images HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1536, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1514, in wsgi_app
    response = self.handle_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/index.py", line 47, in return_images
    if filename.lower().endswith(['.png', '.jpg', '.jpeg', '.gif', '.webp']):
       ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: endswith first arg must be str or a tuple of str, not list
127.0.0.1 - - [23/Jun/2025 13:59:28] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 13:59:28] "[36mGET /static/index.css HTTP/1.1[0m" 304 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 315-855-487
127.0.0.1 - - [23/Jun/2025 14:59:10] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 315-855-487
127.0.0.1 - - [23/Jun/2025 15:53:35] "GET /check HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 15:54:35] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 15:55:18] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 315-855-487
127.0.0.1 - - [23/Jun/2025 16:26:14] "GET /check HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 16:26:16] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [23/Jun/2025 16:27:20] "GET /imageGallery HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 16:27:20] "[35m[1mGET /images HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1536, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1514, in wsgi_app
    response = self.handle_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/index.py", line 47, in return_images
    if filename.lower().endswith(['.png', '.jpg', '.jpeg', '.gif', '.webp']):
       ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: endswith first arg must be str or a tuple of str, not list
127.0.0.1 - - [23/Jun/2025 16:27:36] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 16:32:02] "GET /imageGallery HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 16:32:02] "[35m[1mGET /images HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1536, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1514, in wsgi_app
    response = self.handle_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/index.py", line 47, in return_images
    if filename.lower().endswith(['.png', '.jpg', '.jpeg', '.gif', '.webp']):
       ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: endswith first arg must be str or a tuple of str, not list
127.0.0.1 - - [23/Jun/2025 16:32:03] "GET /imageGallery HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 16:32:03] "[35m[1mGET /images HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1536, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1514, in wsgi_app
    response = self.handle_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/index.py", line 47, in return_images
    if filename.lower().endswith(['.png', '.jpg', '.jpeg', '.gif', '.webp']):
       ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: endswith first arg must be str or a tuple of str, not list
127.0.0.1 - - [23/Jun/2025 16:32:03] "GET /imageGallery HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 16:32:03] "[35m[1mGET /images HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1536, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1514, in wsgi_app
    response = self.handle_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/index.py", line 47, in return_images
    if filename.lower().endswith(['.png', '.jpg', '.jpeg', '.gif', '.webp']):
       ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: endswith first arg must be str or a tuple of str, not list
127.0.0.1 - - [23/Jun/2025 16:32:03] "GET /imageGallery HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 16:32:03] "[35m[1mGET /images HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1536, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1514, in wsgi_app
    response = self.handle_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/index.py", line 47, in return_images
    if filename.lower().endswith(['.png', '.jpg', '.jpeg', '.gif', '.webp']):
       ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: endswith first arg must be str or a tuple of str, not list
127.0.0.1 - - [23/Jun/2025 16:32:03] "GET /imageGallery HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 16:32:03] "[35m[1mGET /images HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1536, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1514, in wsgi_app
    response = self.handle_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/virtualenv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python/index.py", line 47, in return_images
    if filename.lower().endswith(['.png', '.jpg', '.jpeg', '.gif', '.webp']):
       ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: endswith first arg must be str or a tuple of str, not list
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 315-855-487
127.0.0.1 - - [23/Jun/2025 17:57:05] "GET /check HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 17:57:08] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 315-855-487
127.0.0.1 - - [23/Jun/2025 17:57:25] "GET /check HTTP/1.1" 200 -
127.0.0.1 - - [23/Jun/2025 18:00:08] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 18:00:10] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 18:00:12] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 18:00:14] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 18:00:16] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 18:00:18] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 18:00:20] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 18:00:22] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 18:00:24] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [23/Jun/2025 18:00:26] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.07s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 114-643-712
127.0.0.1 - - [25/Jun/2025 18:38:57] "GET /check HTTP/1.1" 200 -
127.0.0.1 - - [25/Jun/2025 18:39:49] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [25/Jun/2025 18:40:24] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [25/Jun/2025 18:40:32] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [25/Jun/2025 18:41:33] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 18:41:35] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 18:41:37] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 18:41:38] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 18:42:06] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 19:13:52] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 19:13:54] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 19:31:56] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 19:37:38] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 19:44:50] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 19:47:02] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 19:47:43] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 19:50:06] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 20:02:20] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 20:11:00] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 20:18:07] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 20:24:08] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 20:32:54] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 20:32:59] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 20:33:12] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 20:33:35] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 20:33:50] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [25/Jun/2025 20:34:02] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.09s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 472-031-606
127.0.0.1 - - [26/Jun/2025 13:24:32] "GET /check HTTP/1.1" 200 -
127.0.0.1 - - [26/Jun/2025 13:41:48] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 13:41:52] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 13:42:32] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 13:42:46] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 13:45:54] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 14:51:13] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 14:52:00] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 14:57:08] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 15:01:20] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 15:02:23] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 15:09:40] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 15:10:50] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 15:22:00] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 15:25:06] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 15:35:53] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 15:35:59] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 16:22:49] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 16:23:57] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 16:24:17] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [26/Jun/2025 16:38:51] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [26/Jun/2025 16:50:45] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [26/Jun/2025 17:08:34] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.08s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 883-434-471
127.0.0.1 - - [27/Jun/2025 14:31:42] "GET /check HTTP/1.1" 200 -
127.0.0.1 - - [27/Jun/2025 14:32:53] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [27/Jun/2025 14:33:02] "GET /images/Cloro.jpeg HTTP/1.1" 200 -
127.0.0.1 - - [27/Jun/2025 14:33:08] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [27/Jun/2025 14:43:04] "GET /images/Cloro.jpeg HTTP/1.1" 200 -
127.0.0.1 - - [27/Jun/2025 15:51:48] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [27/Jun/2025 16:04:53] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [27/Jun/2025 16:05:04] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [27/Jun/2025 16:07:32] "[36mGET /images/Cloro.jpeg HTTP/1.1[0m" 304 -
127.0.0.1 - - [27/Jun/2025 16:20:24] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
127.0.0.1 - - [27/Jun/2025 16:22:20] "GET /check HTTP/1.1" 200 -
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.3.5, pluggy-1.6.0
rootdir: /home/archy/Workspace/Backup/Codes/Golang/TrabalhoTCC/python
collected 2 items

tests/test_index.py ..                                                   [100%]

============================== 2 passed in 0.04s ===============================
Os testes passaram. Iniciando o servidor.
 * Serving Flask app 'index'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
